src/MoBi.Presentation/Presenter/ParameterValuesPresenter.cs

```diff
@@ -119,7 +119,8 @@ private IMoBiCommand addParameterValuesForObjectPaths(IReadOnlyList<ObjectPath>
          if (allSkipped.Any())
             _dialogCreator.MessageBoxInfo(AppConstants.Captions.BuildingBlockAlreadyContains(allSkipped.Select(x => x.PathAsString).ToList()));
 
-         _dialogCreator.MessageBoxInfo(AppConstants.Warnings.CheckDimensionOfAddedParameters);
+         if(allSkipped.Any() || objectPathsToAdd.Any())
```

Comment:
This will mean that if we skip all paths the prompt will still appear, but I think it should be prompt only if any object paths are added.

If all paths are skipped, then no prompt.

So you only need `if(objectPathsToAdd.Any())`

Reply:
oh, could you also remove the bold, I think italic only is the normal convention